src/usePerpsGetMarketSummary.ts

```diff
@@ -2,20 +2,23 @@ import { useQuery } from '@tanstack/react-query';
 import type { ethers } from 'ethers';
 import { fetchPerpsGetMarketSummary } from './fetchPerpsGetMarketSummary';
 import { fetchPerpsGetMarketSummaryWithPriceUpdate } from './fetchPerpsGetMarketSummaryWithPriceUpdate';
+import { useAllPriceFeeds } from './useAllPriceFeeds';
 import { useErrorParser } from './useErrorParser';
 import { useImportContract } from './useImports';
 import { usePriceUpdateTxn } from './usePriceUpdateTxn';
 import { useSynthetix } from './useSynthetix';
 
 export function usePerpsGetMarketSummary({
   provider,
-  priceIds,
   perpsMarketId,
-}: { provider?: ethers.providers.BaseProvider; priceIds?: string[]; perpsMarketId: ethers.BigNumber }) {
+}: { provider?: ethers.providers.BaseProvider; perpsMarketId: ethers.BigNumber }) {
   const { chainId } = useSynthetix();
+  const { data: priceIds } = useAllPriceFeeds();
```

Comment:
Looks like we dont need priceIds in this mutation

Reply:
yes, removed.